deploy/docker-update-stadsdelen.sh

```diff
@@ -4,8 +4,4 @@ set -u   # crash on missing env variables
 set -e   # stop on any error
 set -x   # print what we are doing
 
-/deploy/docker-wait.sh
```

Comment:
Moeten we hier niet checken of de db er is?

Reply:
dat is verplaatst naar de docker-compose:
```    entrypoint: /deploy/wait-for-it.sh database:5432 --
    command: deploy/docker-update-stadsdelen.sh
```